collector.go

```diff
@@ -669,6 +669,10 @@ func indexOidsAsString(indexOids []int, typ string, fixedSize int, implied bool)
 			parts[i] = o
 		}
 		return fmt.Sprintf("%02X%02X:%02X%02X:%02X%02X:%02X%02X:%02X%02X:%02X%02X:%02X%02X:%02X%02X", parts...), subOid, indexOids
+	case "EnumAsInfo":
+		subOid, indexOids := splitOid(indexOids, 1)
+		return enumValues[subOid[0]], subOid, indexOids
```

Comment:
This should have a fallback in case the value isn't within enumValues.

Reply:
Got it. What kind of fallback do you suggest? Maybe just the value of `suboid[0]`?

Reply:
Yes, that's what we usually do.